currency_exchanger/currency_exchanger/request.py

```diff
@@ -0,0 +1,14 @@
+import requests
+from secrets import API_TOKEN
+
+url = "https://api.apilayer.com/exchangerates_data/latest?symbols=USD,BRL,EUR,BTC&base=USD"
+
+payload = {}
+headers= {
+  "apikey": API_TOKEN
+}
+
+response = requests.request("GET", url, headers=headers, data = payload)
+
```

Comment:
Shouldn't this code be a separate function with a constant URL taken from some file like secrets.py? It makes it callable from other files

`response = requests.request("GET", secrets.BASE_CURRENCY_URL, headers=headers, data = payload)`